### ISSUE TITLE
MobilenetV3 Top 5 Accuracy issue

### ISSUE THREAD

**Issue by ConcentrativeMan** [tensorflow/tensorflow#42682]
I use pre-trained model from this site: https://github.com/tensorflow/models/tree/master/research/slim/nets/mobilenet 
I download v3-large_224_1.0_float.pb model link is (Large dm=1 (float) ). After I run it, I often get this result,
===== TENSORFLOW RESULTS =======
cello, violoncello (score = 1.00000)
hip, rose hip, rosehip (score = 0.00000)
banjo (score = 0.00000)
steel drum (score = 0.00000)
marimba, xylophone (score = 0.00000)
Then, I change input image, but the result is the same. I do know how to solve this problem.Thank you.
My code is as below:
[mobilenet_v3.txt](https://github.com/tensorflow/tensorflow/files/5131057/mobilenet_v3.txt)

**Comment by Saduf2019** [comment]
@jiaqinghao2009 
Please fill in the issue template for us to replicate the issue faced, we would need the tf version, simple stand alone indented code, system information or if possible share a colab gist with the issue reported and error logs.

[You could use this [link](https://colab.sandbox.google.com/#create=true&language=python3) to run the code and share the Gist (File -> Save a copy as Github gist) with us. Thanks!]

**Comment by jiaqinghao2009** [comment]
Sorry, I can not open your link. My tensorflow version is 1.14.0., Ubuntu 18.04, llvm 10.0. I post the same question at [https://stackoverflow.com/questions/63607979/mobilenetv3-top-5-accuracy-issue]. 
When I run mobilenetv2, I only change two line code. 
1. model_path = os.path.join(pwd, 'model/mobilenet_v2_1.4_224_frozen.pb') 
2. softmax_tensor = sess.graph.get_tensor_by_name('MobilenetV2/Predictions/Softmax:0')
 Result is as below: 
===== TENSORFLOW RESULTS ======= 
Tibetan mastiff (score = 0.14538)
ice cream, icecream (score = 0.09276) 
alligator lizard (score = 0.02763) 
water tower (score = 0.02139) 
slot, one-armed bandit (score = 0.01965)
I wonder why mobilenetv3 get a 1.00000 score four 0.00000 score.
p.s. Code related with tvm can be deleted(I forget delete this code.)

**Comment by Saduf2019** [comment]
@ConcentrativeMan 

We will need the code used to replicate the issue, or a colab gist for us to ananlyse.
